frontend/src/pages/profile/[id].tsx

```diff
@@ -17,13 +17,15 @@ const UserProfile: NextPage = (): JSX.Element => {
   const { user: author } = useAuth({ middlware: 'auth' }) 
   const { id } = router.query
 
-  const { data: user, mutate } = useSWR(`/api/user-privilege/${id}`, async () => fetcher(`/api/user-privilege/${id}`), {
+  const { data, mutate } = useSWR(`/api/user-privilege/${id}`, fetcher, {
     refreshInterval: 1000,
     revalidateOnMount: true
   })
+  const user = data?.data?.user
+  const activities = data?.data?.activities
```

Comment:
destructure inner `data`

Reply:
Do you have any suggestion about destructuring sir?

![image](https://user-images.githubusercontent.com/108642414/187396978-ef305088-bb5c-4d61-ae20-492d792e6944.png)
![image](https://user-images.githubusercontent.com/108642414/187397128-289b8552-aa30-420b-8ed2-3107c2cb071e.png)